### PR TITLE
Improve Validator docs

### PR DESCRIPTION
**Pull request by DrW3RK**
- Improve the Kusama Validator doc
- Add preliminary info about 1KV program
- remove outdated links/content

### CODE REVIEW

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 1]
```diff
@@ -7,15 +7,93 @@ keywords: [validate, validator, kusama, stake, maintain]
...
+the existential deposit, plus a little extra for transaction fees. The rest can come from
+nominators. To understand how validators are elected, check the
+[NPoS Election algorithms](../../learn/learn-phragmen.md) page.
+
+:::info On-Chain Data for Reference
+
+On Polkadot, the minimum stake backing a validator in the active set is
+{{ polkadot: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :polkadot }}
+{{ kusama: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :kusama }} in the era
+{{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :polkadot }}
```
Comment (filippoweb3) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
{{ kusama: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :kusama }} in the era {{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :polkadot }}
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 2]
```diff
@@ -7,15 +7,93 @@ keywords: [validate, validator, kusama, stake, maintain]
...
+2. Further decentralize the validator active set.
+
+[Apply to Thousand Validator Program on Kusama](https://thousand-validators.kusama.network/#/)
+
+## Run a Kusama Validator
+
+Running a validator on the Kusama network is identical to running a Polkadot validator.
+
+Make sure to adjust the Polkadot guide to run a Kusama network validator (The instructions will also
+available in the Polkadot Validator guide):
```
Comment (filippoweb3) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md]:
```suggestion
Make sure to adjust the Polkadot guide to run a Kusama network validator (the instructions will also be available in the Polkadot Validator guide):
```

File: docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md [hunk 3]
```diff
@@ -7,15 +7,93 @@ keywords: [validate, validator, kusama, stake, maintain]
...
+{{ polkadot: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :polkadot }}
+{{ kusama: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :kusama }} in the era
+{{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :polkadot }}
+{{ kusama: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :kusama }}
+
+On Kusama, the minimum stake backing a validator in the active set is
+{{ kusama: <MinimumStake network="kusama" defaultValue={5288388652143741} /> :kusama }}
+{{ polkadot: <MinimumStake network="kusama" defaultValue={5288388652143741} /> :polkadot }} in the
+era
+{{ kusama: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4838"/>. :kusama }}
```
Comment (filippoweb3) [inline docs/maintain/kusama/maintain-guides-how-to-validate-kusama.md:59]:
```suggestion
era {{ kusama: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4838"/>. :kusama }}
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 4]
```diff
@@ -66,17 +54,17 @@ understand how validators are elected, check the
 :::info On-Chain Data for Reference
 
 On Polkadot, the minimum stake backing a validator in the active set is
-{{ polkadot: <MinimumStake network="polkadot" defaultValue={18684315524834056}/> :polkadot }}
-{{ kusama: <MinimumStake network="polkadot" defaultValue={18684315524834056}/> :kusama }} in the era
-{{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="799"/>. :polkadot }}
-{{ kusama: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="799"/>. :kusama }}
+{{ polkadot: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :polkadot }}
+{{ kusama: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :kusama }} in the era
+{{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :polkadot }}
```
Comment (filippoweb3) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md:67]:
```suggestion
{{ kusama: <MinimumStake network="polkadot" defaultValue={17314855524834056}/> :kusama }} in the era {{ polkadot: <RPC network="polkadot" path="query.staking.currentEra" defaultValue="998"/>. :polkadot }}
```

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md [hunk 5]
```diff
@@ -66,17 +54,17 @@ understand how validators are elected, check the
...
 
 On Kusama, the minimum stake backing a validator in the active set is
-{{ kusama: <MinimumStake network="kusama" defaultValue={5367388652143741} /> :kusama }}
-{{ polkadot: <MinimumStake network="kusama" defaultValue={5367388652143741} /> :polkadot }} in the
+{{ kusama: <MinimumStake network="kusama" defaultValue={5288388652143741} /> :kusama }}
+{{ polkadot: <MinimumStake network="kusama" defaultValue={5288388652143741} /> :polkadot }} in the
 era
-{{ kusama: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4058"/>. :kusama }}
-{{ polkadot: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4058"/>. :polkadot }}
+{{ kusama: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4838"/>. :kusama }}
```
Comment (filippoweb3) [inline docs/maintain/maintain-guides-how-to-validate-polkadot.md:74]:
```suggestion
era {{ kusama: <RPC network="kusama" path="query.staking.currentEra" defaultValue="4838"/>. :kusama }}
```

**Review by filippoweb3 — COMMENTED** [review]
lgtm, some minor things when displaying the era numbers (space missing)

**Review by filippoweb3 — APPROVED** [review]

File: docs/maintain/maintain-guides-how-to-validate-polkadot.md (hunk 5 above)
Comment (profradha) [reply docs/maintain/maintain-guides-how-to-validate-polkadot.md:74]: I'm unable to commit this!

**Comment by michalisFr** [comment]
With the exception of some links, references to DOT and the obvious repetition of information, overall this looks good. 

I would emphasise the following things a bit more:
- Suggest that running a validator on Kusama before running one on Polkadot is recommended
- 1KV can be segued from the previous section and mention that it's there to help bootstrap community validators while gathering nominations on their own and it's a prerequisite that you run a Kusama validator first.
- Instead of linking to the 1KV dashboard you should link to the article:
https://wiki.polkadot.network/docs/thousand-validators
- Other differences to be mentioned:
1. Min commission on Kusama is 10% (can be changed through governance)
2. 4 eras per day as opposed to 1 for Polkadot

**Comment by DrW3RK** [comment]
Thank you @michalisFr Will incorporate the feedback!

**Pull request merged**